samples/bookinfo/kube/rbac/rbac-config-ON.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: "config.istio.io/v1alpha2"
+kind: RbacConfig
+metadata:
+  name: default
+spec:
+  mode: 'ON'
```

Comment:
Shall we enable it only on "default" namespace?

Reply:
Yeah, that probably better for user to try, updated.